tough/src/transport.rs

```diff
@@ -139,6 +139,7 @@ impl Error for TransportError {
 pub struct FilesystemTransport;
 
 impl Transport for FilesystemTransport {
+    #[cfg(not(target_os = "windows"))]
```

Comment:
```suggestion
```

I'm working on getting a Windows machine setup, so I'm not sure if this works yet. But if we use the URL function [to_file_path](https://docs.rs/url/2.2.0/url/struct.Url.html#method.to_file_path) might work for us.

Could you try changing line 151 to:

`let f = std::fs::File::open(url.to_file_path().unwrap()).map_err(|e| {`

and removing the other changes to see if that works?
If it works we will want to handle the error instead of `unwrap()`, but we can see if it works first.

Reply:
Ok , trying it now

Reply:
It looks like it works ! (omg I'm sorry I haven't thought about this one...)

Reply:
Nice!